frontend/src/Pages/DebateRoom.tsx

```
@@ -217,6 +217,8 @@ const extractJSON = (response: string): string => {
 };
 
 const DebateRoom: React.FC = () => {
+  const judgingRef = useRef(false);
+  const navigate = useNavigate();
   const location = useLocation();
   const navigate = useNavigate();
```

Biome (2.1.2)

[error] Shouldn't redeclare 'navigate'. Consider to delete it or rename it.

'navigate' is defined here:


(lint/suspicious/noRedeclare)